internal/util/testutil/equal_test.go

```diff
@@ -54,3 +56,80 @@ func TestEqual(t *testing.T) {
 		time.Date(2022, time.March, 11, 8, 8, 42, 123456789, time.UTC),
 	)
 }
+
+func TestEqualValue(t *testing.T) {
```

Comment:
So, with this test we check that `EqualValue` works correctly, right? I see codecov shows some threads in `equal.go` function as not covered, is it on purpose?

Reply:
This was removed.